src/main/Metadata.tsx

```diff
@@ -278,7 +284,19 @@ const Metadata: React.FC = () => {
    * Validator for required fields
    * @param value
    */
-  const required = (value: string) => (value ? undefined : t("metadata.validation.required"));
+  const required = (value: unknown) => {
+    let val = value;
+
+    if (value && typeof value === "object" && "submitValue" in value) {
+      val = value.submitValue;
+    }
+
+    if (value && Array.isArray(value) && value.length === 0) {
+      val = false;
+    }
```

Comment:
In case you fill out `mixed_text` metadata value and delete the content afterwords, the required message isn't shown as expected. The value is `['']` in this case. Lets get rid of empty strings from array before checking the length.

```suggestion
    if (value && Array.isArray(value) && value.filter(v => v).length === 0) {
      val = false;
    }
```

Reply:
Can't reproduce this.

[Bildschirmaufzeichnung vom 2025-05-28 10-08-07.webm](https://github.com/user-attachments/assets/96a7104a-0de9-48fc-925e-186a17c0c7ff)

Could you tell me what I'm doing wrong, or have any other tips for me on how to reproduce this? I worry that empty strings in the array point at a more fundamental problem.

Reply:
Sure, here is an example

[editor-required-metadata-field-on-focus-lost.webm](https://github.com/user-attachments/assets/3cd6ef33-02fe-4bad-b8ba-568d9d520a50)

The affected metadata field configuration looks like

![image](https://github.com/user-attachments/assets/fd7a343a-3183-4894-bdaf-f49229248d27)

Reply:
The delimiter configuration is not applying too. But this is a separate issue.

Reply:
Thanks for the detailed report, I forgot to consider that `mixed_text` fields may not have a listprovider defined. The latest commit 74c72b should fix the issue you were seeing.

I agree that respecting delimiter configuration is better handled in a separate issue/PR.

Reply:
Yes, thanks.